### PR TITLE
ssl-config: 0.6.0 -> 0.6.1

### PR DESCRIPTION
**Pull request by raboof**
To test out https://github.com/lightbend/ssl-config/pull/342

### CODE REVIEW

**Comment by raboof** [comment]
> For https://github.com/akka/akka/pull/31046 all checks passed as well, so how do you actually verify if an updated dependency actually is (not) binary compatible with its previous version?

MiMa only checks the API's of the project itself, not of the dependencies - so indeed there is no automated check when doing the update here. We mostly rely on the checks (and, if we choose to add mima filters there, the judgement when adding those filters) in the projects we depend on, and common sense ;)

**Comment by mkurz** [comment]
@raboof All checks pass :+1: Also maybe cut an actual _GitHub_ release for ssl-config 0.6.1? See https://github.com/lightbend/ssl-config/releases

File: project/AkkaBuild.scala
Comment (mkurz) [inline project/AkkaBuild.scala:87]: @raboof Don't forget to revert this before merging :wink:

**Review by patriknw — APPROVED** [review]
LGTM after reverting the snapshot repo

**Comment by raboof** [comment]
> @raboof All checks pass +1 Also maybe cut an actual _GitHub_ release for ssl-config 0.6.1? See https://github.com/lightbend/ssl-config/releases

:+1: https://github.com/lightbend/ssl-config/releases/tag/v0.6.1

**Pull request merged**
